README.md

```diff
@@ -1,18 +1,21 @@
-# polkadot{.js} extension
+# Signer
 
-A very simple scaffolding browser extension that injects a [@polkadot/api](https://github.com/polkadot-js/api) Signer into a page, along with any associated accounts, allowing for use by any dapp. This is an extensible POC implementation of a Polkadot/Substrate browser signer.
+A browser extension managing accounts and signing transactions in a secure way. It's a fork of [polkadot-js/extension](https://github.com/polkadot-js/extension). Extension enables simple integration with dapps using [polkadot-js/extension-dapp](https://polkadot.js.org/docs/extension/) API.
 
-As it stands, it does one thing: it _only_ manages accounts and allows the signing of transactions with those accounts. It does not inject providers for use by dapps at this early point, nor does it perform wallet functions where it constructs and submits txs to the network.
+Working with (among others):
+* https://azero.dev
+* https://dashboard.azero.dev
+* https://polkadot.js.org/apps
 
-## Installation
 
-- On Chrome, install via [Chrome web store](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd)
-- On Firefox, install via [Firefox add-ons](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/)
+## Installation
 
-![interface screenshots](docs/extension-overview.png)
+- On Chrome, install via [Chrome web store](TODO)
+- On Firefox, install via [Firefox add-ons](TODO)
 
 ## Documentation and examples
-Find out more about how to use the extension as a Dapp developper, cookbook, as well as answers to most frequent questions in the [Polkadot-js extension documentation](https://polkadot.js.org/docs/extension/)
+
+Simple Singer integration guide `<add link here>`. Find more information in [Polkadot-js extension documentation](https://polkadot.js.org/docs/extension/)
```

Comment:
> add link here

Just a reminder to not forget about it.

Reply:
Oh, and probably a link to replace with our own: `Polkadot-js extension documentation`? Or are we keeping the polkadot's one?

Reply:
```suggestion
Simple Singer integration guide `<add link here>`. Find more information in [polkadot-js/extension-dapp API documentation](https://polkadot.js.org/docs/extension/)
```
For consistency with the intro?